x-pack/plugins/osquery/public/scheduled_query_groups/queries/use_scheduled_query_group_query_form.tsx

```diff
@@ -34,8 +36,28 @@ export interface ScheduledQueryGroupFormData {
 export const useScheduledQueryGroupQueryForm = ({
   defaultValue,
   handleSubmit,
-}: UseScheduledQueryGroupQueryFormProps) =>
-  useForm<OsqueryManagerPackagePolicyConfigRecord, ScheduledQueryGroupFormData>({
+}: UseScheduledQueryGroupQueryFormProps) => {
+  const { data } = useScheduledQueryGroups();
+  const ids = useMemo<string[]>(
+    () =>
+      data?.items
+        .map((it) =>
+          it.inputs.map((input) => input.streams.map((stream) => stream.compiled_stream.id)).flat()
```

Comment:
`id` should be unique only in the context of the same agent policy, because if it will be unique globally then the user will be not able to correlate results from the same "pack" across all the agents